examples/data_view_field_editor_example/public/app.tsx

```diff
@@ -75,22 +82,41 @@ const DataViewFieldEditorExample = ({ dataView, dataViewFieldEditor }: Props) =>
     },
   ];
 
+  const preconfigureId = useGeneratedHtmlId({ prefix: 'usePreconfigured' });
   const content = dataView ? (
     <>
       <EuiText data-test-subj="dataViewTitle">Data view: {dataView.title}</EuiText>
-      <div>
-        <EuiButton
-          onClick={() =>
-            dataViewFieldEditor.openEditor({
-              ctx: { dataView },
-              onSave: refreshFields,
-            })
-          }
-          data-test-subj="addField"
-        >
-          Add field
-        </EuiButton>
-      </div>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            onClick={() =>
+              dataViewFieldEditor.openEditor({
+                ctx: { dataView },
+                onSave: refreshFields,
+                fieldToCreate: preconfigured
+                  ? {
+                      name: 'demotestfield',
+                      type: 'boolean',
+                      script: { source: 'emit(true)' }, // optional
+                      customLabel: 'cool demo test field', // optional
+                      format: { id: 'boolean' }, // optional
+                    }
+                  : undefined,
+              })
+            }
+            data-test-subj="addField"
+          >
+            Add field
+          </EuiButton>
+        </EuiFlexItem>
+        <EuiFlexItem data-test-subj="preconfiguredControlWrapper">
+          <EuiCheckbox
```

Comment:
When I click on this checkbox it never visually gets into "checked" state [x], it always stays unchecked [] . probably need to add a value prop

<img width="480" alt="Screenshot 2022-07-22 at 11 06 20" src="https://user-images.githubusercontent.com/7784120/180405131-2d031093-4df8-49c9-9265-5b825ae4ea3d.png">

Reply:
Thanks for catching this! It's due to a last-minute markup change to get the functional tests working. This should be fixed in 1b24c0e7d25